.changeset/good-rocks-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/i18n": patch
```

Comment:
Shall we bump console and myaccount as well?

Reply:
+1
- For i18n change to be reflected in console, console patch is required.
- Since myaccount depends on i18n, let's bump that as well

Reply:
Addressed https://github.com/wso2/identity-apps/pull/4819/commits/2c9715c4dec09200c65750e62602a725cebf5b2e